repos.yml

```
@@ -1,4 +1,8 @@
 repos:
+  - repo: aapt
```

```suggestion
  - repo: android-platform-frameworks-base
```

然后 info 改成描述这个包的信息就可以了。